.eslintrc.js

```diff
@@ -1,10 +1,14 @@
 module.exports = {
+  globals: {
+    server: true
```

Comment:
Is this needed? As far as i can see you dont use the mirage global anywhere.

Reply:
Nope, you're right!